File: web/src/entity/rider.ts
/**
 * 专业
 * author: liguowen
 */
import {User} from './user';

export class Rider {
  /**
   * id
   */
  id?: number;

  /**
   * 用户
   */
  user?: User;

  status?: number;

  phone?: number;

  constructor(data = {} as {
    id?: number,
    user?: User,
    status?: number,
    phone?: number,

  }) {
    this.id = data.id;
    this.user = data.user;
    this.status = data.status;
    this.phone = data.phone;
  }
}

File: api/src/main/java/club/yunzhi/log/entity/Rider.java
package club.yunzhi.log.entity;


import lombok.Getter;
import lombok.Setter;

import javax.persistence.*;

@Entity
@Setter
@Getter
public class Rider {
    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private Long id;

    private Long phone;

    private Long status;

    @OneToOne
    private User user;
}
